airbyte-integrations/connectors/source-pipedrive/source_pipedrive/schemas/activity_fields.json

```diff
@@ -0,0 +1,268 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "The root schema",
+  "description": "The root schema comprises the entire JSON document.",
+  "default": {},
+  "examples": [
+    {
+      "id": 1,
+      "key": "title",
+      "name": "Title",
+      "order_nr": 2,
+      "field_type": "varchar",
+      "add_time": "2019-02-04 13:58:03",
+      "update_time": "2019-02-04 13:58:03",
+      "last_updated_by_user_id": 1,
+      "active_flag": true,
+      "edit_flag": false,
+      "index_visible_flag": true,
+      "details_visible_flag": true,
+      "add_visible_flag": true,
+      "important_flag": false,
+      "bulk_edit_allowed": true,
+      "searchable_flag": false,
+      "filtering_allowed": true,
+      "sortable_flag": true,
+      "options": [
+        {
+          "id": 190,
+          "label": "Private person"
+        }
+      ],
+      "mandatory_flag": true
+    }
+  ],
+  "required": [
+    "id",
+    "key",
+    "name",
+    "order_nr",
+    "field_type",
+    "add_time",
+    "update_time",
+    "last_updated_by_user_id",
+    "active_flag",
+    "edit_flag",
+    "index_visible_flag",
+    "details_visible_flag",
+    "add_visible_flag",
+    "important_flag",
+    "bulk_edit_allowed",
+    "searchable_flag",
+    "filtering_allowed",
+    "sortable_flag",
+    "options",
+    "mandatory_flag"
+  ],
+  "properties": {
+    "id": {
+      "$id": "#/properties/id",
+      "type": "integer",
+      "title": "The id schema",
+      "description": "An explanation about the purpose of this instance.",
+      "default": 0,
+      "examples": [1]
```

Comment:
Also, since at the top of each schema for each stream there is `examples` field that contains example record, why each field in `properties` still contains `examples` field?